src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandOddsRatio.java

```diff
@@ -88,8 +88,8 @@ static double calculateSOR(final int[][] table) {
 
         final double ratio = (t00 / t01) * (t11 / t10) + (t01 / t00) * (t10 / t11);
 
-        final double refRatio = min(t00, t01)/ max(t00, t01);
-        final double altRatio = min(t10, t11)/ max(t10, t11);
+        final double refRatio = (double) min(t00, t01)/ max(t00, t01);
```

Comment:
compiler does not like these 2 changes - min and max return double so this is double and thus the cast is redundant. Can you remove those 2 changes in StrandOddsRatio.java?